http_service/bugbug_http/app.py

```diff
@@ -720,6 +744,191 @@ def batch_prediction(model_name):
     return compress_response({"bugs": data}, status_code)
 
 
+@application.route("/<model_name>/predict/broken_site_report/batch", methods=["POST"])
+@cross_origin()
```

Comment:
I've named the endpoint `broken_site_report` to match the name of the component / telemetry ping https://github.com/mozilla/gecko-dev/blob/606cdd34860e42611fceba025314864cd293a43b/toolkit/components/reportbrokensite/pings.yaml#L8